.github/ISSUE_TEMPLATE/report-an-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Report an issue
+about: Help us improve our service by reporting issues you are experiencing.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what is the issue.
+
+**Expected result vs actual result**
+
+**When did the issue occur?**
+Be as accurate as you can.
+
+**What package did you attempt to install, and from which repo?**
```

Comment:
Suggestion: ``If applicable, what package did you attempt to install, and from which repo?``